src/html/components/buttons.html

```diff
@@ -0,0 +1,22 @@
+<h2>Buttons</h2>
+
+<a class="Btn Btn-Primary" href="#">Primary Button</a>
+<a class="Btn Btn-Secondary" href="#">Secondary Button</a>
+<a class="Btn Btn-Tertiary" href="#">Tertiary Button</a>
+
+<br/ >
+
+<input type="submit" class="Btn Btn-Primary" value="Primary Button" />
+<button class="Btn Btn-Secondary">Secondary Button</button>
+<a class="Btn Btn-Transparent" href="#">Plain button</a>
```

Comment:
Perhaps either make separate rows for each element (eg row of links, row of buttons, row of inputs) with a header above each one, or just keep this row which demonstrates mixed elements and add the transparent one here. Hope that makes sense?